intelmq_manager/static/js/configs.js

```diff
@@ -247,6 +247,16 @@ function save_data_on_files() {
         }
     }
 
+    let token = sessionStorage.getItem("login_token");
+    if (token !== null) {
+        $.ajaxSetup({
+           beforeSend: function (xhr)
+           {
+             xhr.setRequestHeader("Authorization", token);        
+           }
+        });
+    }
+
     Promise.all([
         $.post(API + '/save?file=runtime', generate_runtime_conf(app.nodes))
```

Comment:
Hi, I do not understand much the difference between this `$.ajaxSetup.beforeSend` and `authenticatedAjax` method in `static.js`. It seems to me the code is duplicated now. IMHO it would be better to use `authenticatedAjax` instead of `$.post` here like this:

```js
Promise.all([
  authenticatedAjax( {"url": API + '/save?file=runtime', "data": generate_runtime_conf(app.nodes)})
  .done(saveSucceeded)
  ...

# old
# Promise.all([
#        $.post(API + '/save?file=runtime', generate_runtime_conf(app.nodes))
#            .done(saveSucceeded)
```

As stated in the first line of the [docs](https://api.jquery.com/jquery.ajaxsetup/), the use of `$.ajaxSetup` is not recommended. Here, we change the default behaviour of the ajax request – those requested before saving would look other way than those after saving. Moreover, when saved multiple times, multiple `Authorization` requests would be added, it seems to me.